pkgs/applications/science/electronics/kicad/packages/kikit.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, bc
+, zip
+, python3
+, package
+, thirdPartyPackagePath
+}:
+let
+  python = python3.withPackages (ps: with ps; [ click ]);
+  kikit = python3.pkgs.kikit;
```

Comment:
This should probably live in the python env

Reply:
Are you referring to the python env defined on line 9? If so, that wouldn't work: that python env is only used at build time to run some build scripts invoked by the Makefile in the kikit repo.

We need this kikit python dependency while kicad runs. kicad defines its own python interpreter that eventually imports the python code we're packaging here. That interpreter is defined in the kicad derivation.

Reply:
Marking resolved. Please reopen if you have any concerns.